dotCMS/src/main/java/com/dotmarketing/portlets/links/action/EditLinkAction.java

```diff
@@ -739,7 +741,7 @@ public void _getVersionBackWebAsset(ActionRequest req, ActionResponse res,
 			if (cont.isWorking()) {
 				com.dotmarketing.portlets.contentlet.model.Contentlet newFormatContentlet = 
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "newFormatContentlet". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "newFormatContentlet" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)